package.json

```diff
@@ -5,18 +5,27 @@
   "devDependencies": {
     "babel-eslint": "^7.2.3",
     "eslint": "^4.2.0",
-    "eslint-plugin-babel": "^4.1.1",
     "eslint-config-fbjs": "0.0.0",
+    "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-flowtype": "^2.35.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.1.0",
-    "eslint-plugin-relay": "0.0.8"
+    "eslint-plugin-relay": "0.0.8",
+    "prettier": "^1.14.3"
   },
   "scripts": {
+    "prettier": "find . -name node_modules -prune -or -name dist -prune -or -name '*.js' -print | xargs prettier --write",
```

Comment:
I assume you ran this too and the only change was signed source?

Reply:
Correct as we have `requirePragma: true`, otherwise there was a lot of reformatting that I at least wanted to do separately if desired.